Foundation/NSURL.swift

```diff
@@ -610,8 +610,28 @@ open class NSURL : NSObject, NSSecureCoding, NSCopying {
     */
     /// - Experiment: This is a draft API currently under consideration for official import into Foundation as a suitable alternative
     /// - Note: Since this API is under consideration it may be either removed or revised in the near future
-    open func checkResourceIsReachable() throws -> Bool {
-        NSUnimplemented()
+    open func checkResourceIsReachableAndReturnError(_ error: UnsafeMutablePointer<NSError?>?) -> Bool {
```

Comment:
Did you put back the error argument here for compatibility with Darwin? There it's an autoreleased error, which we don't have on Linux. For this method I'm ok with the divergent API for now.